programas.html

```diff
@@ -1,16 +1,45 @@
 ---
 layout: default
 ---
+<head>
+	<style> 
+	div.cities {
```

Comment:
Why this class is called cities? Shouldn't be like "spaces" or "programs-paragraphs"?